diameter-stack/build.gradle

```diff
@@ -2,8 +2,77 @@ plugins {
   id "org.jetbrains.kotlin.jvm" version "1.2.30"
   id "idea"
   id "java-library"
+  id "signing"
+  id "maven"
```

Comment:
'maven' is added at the parent level, so it "might" be optional here. But, I am not sure.